index.js

```diff
@@ -14,13 +14,14 @@ const defaults = {
   reporterDefaults: {
     filter: [],
     exclude: []
-  }
+  },
+  rateLimit: {}
```

Comment:
so move into reporter defaults.  also should be a number and maybe a rateLimitBasedOnTags: true